pkg/utils/headers.go

```diff
@@ -36,8 +36,8 @@ var (
 	// Then the SDK will set them as ce- headers when sending them through HTTP. Otherwise, when using replies we would
 	// duplicate ce- headers.
 	forwardPrefixes = []string{
-		// knative
-		"knative-",
+		"knative-", // Knative
+		"x-b3-",    // Zipkin (Istio) B3
```

Comment:
Generally OK with this change!

Not sure on the background why that was never done before? I also briefly recall some work on `b3` tracing by Francesco or @pierDipi on the `eventing-kafka-broker`.

/lgtm
/hold 
(for feedback)

Reply:
This issue https://github.com/knative-sandbox/eventing-kafka-broker/issues/725 mentions that Knative Eventing supports b3 headers but I've never verified it.

I think Knative Eventing supports b3 tracing headers only when there is no MT channel `Broker` in the pipeline since the ingress service filters them out.

This patch fixes that, so
/lgtm

Reply:
/approve
/unhold

Reply:
thanks for clarification